nemo_skills/inference/eval/compute_eval.py

```
@@ -0,0 +1,102 @@
+        return
+
+    def setup_llm(self):
+        return
+
+    def setup_litellm_cache(self):
+        return
+
+    def cleanup_litellm_cache(self):
+        return
+
+    async def process_single_datapoint(self, data_point, data):
```

Ruff (0.14.8)

Unused method argument: `data`

(ARG002)